bin/probot-run.js

```diff
@@ -20,6 +20,9 @@ program
 if (!program.app) {
   console.warn('Missing GitHub App ID.\nUse --app flag or set APP_ID environment variable.')
   program.help()
+} else if (!program.webhookProxy) {
```

Comment:
You don’t need the `else` here, `program.help()` ends the process. I’d put `if (!program.webhookProxy) {` in the next line